src/urllib3/connection.py

```diff
@@ -658,11 +667,19 @@ def connect(self) -> None:
         self.sock = sock_and_verified.socket
         self.is_verified = sock_and_verified.is_verified
 
+        # TODO reword
```

Comment:
I'm not sure how this is related to the issue

Reply:
I had difficulties to understand the documentation. It should be revised (can do it once i have more clarity, potentially during work on #3267)

Reply:
=> clarified a bit more

Reply:
=> removed this comment